src/NServiceBus.Core/Recoverability/Faults/MoveFaultsToErrorQueueBehavior.cs

```diff
@@ -6,23 +6,45 @@ namespace NServiceBus
     using NServiceBus.Logging;
     using NServiceBus.Pipeline;
     using NServiceBus.Pipeline.Contexts;
+    using NServiceBus.Recoverability.Faults;
     using NServiceBus.Routing;
     using NServiceBus.TransportDispatch;
     using NServiceBus.Transports;
 
     class MoveFaultsToErrorQueueBehavior : Behavior<TransportReceiveContext>
```

Comment:
I think that FLR, SLR and Faults behaviors could use refactoring #3047. My current idea is to merge those three together but that definitely needs a discussion.